nlp_worker/annotate/tasks.py

```diff
@@ -0,0 +1,29 @@
+import os
+import json
+import logging
+
+from utils.celery_client import celery_app
+from worker.tasks import BaseTask
+
+from nlp_worker.annotate.annotate import annotate
+
+log = logging.getLogger(__name__)
+
+
+class AnnotateTask(BaseTask):
+
+    name = "annotate"
+
+    def execute_task(self):
+        work_path = self.get_work_path(self.job_id)
+        json_path = os.path.join(work_path, 'text.json')
+        with open(json_path) as data_object:
+            data = json.load(data_object)
+
+        self.update_state(state='PROGRESS',
```

Comment:
Ist an sich eine gute Idee, den Status hier auf `PROGRESS` zu setzen. Machen wir aber in keinem anderen Task so. Wir sollten das mal im Team besprechen. Aber an sich würde ich das dann im BaseTask ansiedeln (wenn Celery das nicht schon selbst mitbringt). Also würde ich es hier erstmal rausnehmen.